src/components/RestaurantDetails/components/Filters/Filters.js

```diff
@@ -0,0 +1,29 @@
+//Libraries
+import React from 'react';
+
+//Icons
+import heartIcon from '../../../../images/heart.svg';
+
+//Components
+import SearchInput from '../../../../commonComponents/SearchInput';
+import Favourite from '../../../../commonComponents/IconWithLabel';
+import Checkbox from '../../../../commonComponents/Checkbox';
+
+//CSS
+import './Filters.css';
+function Filters() {
+  return (
+    <div className='filters'>
+      <SearchInput className='filterItem' />
+      <Checkbox
+        className='filterItem'
+        label='Veg Only'
+        htmlFor='isVeg'
+        isChecked={false}
+      />
+      <Favourite className='filterItem' icon={heartIcon} label='Favourite' />
```

Comment:
As we'd discussed if it is a Favourite component then don't pass the icon or name this component IconWithLabel to make it more generic and then pass the icon prop